### DISCUSSION TITLE
Introducing Veramo ~ A JavaScript Framework for Verifiable Data

### DISCUSSION THREAD

**Discussion by jasheal** [decentralized-identity/veramo#316] [Announcements]
After 18 months of hard work, refactoring, breaking things, and feedback from some great alpha partners, we are excited to announce the branding and soft launch of daf under our new name Veramo! We are now officially in a public beta.

We would like to thank everyone who has helped us to get Veramo to this point. We still have a long way to go, but the foundations have been set for an exciting 2021. We have created a solid foundation on which we can iterate fast with minimal breaking change, created streamlined workflows that will allow us to take more contributions and feedback from the community, and set up a full documentation pipeline that will be kept up-to-date with every new release. 

As we are in public beta, we are interested in talking with potential beta partners to work closely with over the next year. We have limited spots, so if you have a great use case and would like to work closely with us to realise it, please reach out to us here or on hello@veramo.io to discuss it!

~ Veramo Team

**Comment by vlmold** [comment]
great! Congratulations! We have been using daf project for SSI Health wallet and it worked just great!

**Comment by jasheal** [comment]
@vladyslavmunin we would love to see a demo of your app if you can share?

**Reply by vlmold** [reply]
@jasonhealy of course I can share it, what is the best way to reach you? my email is in the profile.

**Reply by jasheal** [reply]
Great! Can you send a video demo or something similar to something to `veramo-hello@mesh.xyz`
